sdk/test-utils/recorder/package.json

```diff
@@ -67,8 +67,8 @@
   "dependencies": {
     "@azure/core-http": "^2.0.0",
     "@azure/core-rest-pipeline": "^1.1.0",
-    "@azure/test-utils": "^1.0.0",
-    "@azure/core-auth": "^1.3.2"
+    "@azure/core-auth": "^1.3.2",
+    "@azure/core-util": "^1.0.0-beta.1"
   },
```

Comment:
@joheredi we can publish `@azure/test-utils`, but it's the naming that concerns me.
We might want to put it under `@azure-tools` scope as the bare minimum.

Reply:
I'm just avoiding that discussion for now.

Reply:
Ultimately I think the right move is to dedupe everything to core-util and ship it